### PR TITLE
Update ja.ts

### PR DESCRIPTION
**Pull request by sj-h4**
#### Description of changes

In `<authenticator>`, 'Enter your Username' is used instead of 'Enter your username', so I added a Japanese translation for 'Enter your Username'.

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)
- [x] `yarn test` passes and tests are updated/added
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/ui/src/i18n/dictionaries/authenticator/ja.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const jaDict: AuthenticatorDictionary = {
   'Enter your code': 'コードを入力',
   'Enter your Email': 'メールアドレスを入力',
   'Enter your phone number': '電話番号を入力',
+  'Enter your Username': 'ユーザー名を入力 ',
```
Comment (esauerbo) [inline packages/ui/src/i18n/dictionaries/authenticator/ja.ts:25]: I may be wrong but this looks the same as the original string, maybe a copy/paste error?

File: packages/ui/src/i18n/dictionaries/authenticator/ja.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ export const jaDict: AuthenticatorDictionary = {
...
   'Enter your username': 'ユーザー名を入力 ',
```
Comment (esauerbo) [inline packages/ui/src/i18n/dictionaries/authenticator/ja.ts:26]: Could you update the existing translation instead of creating a separate one?

**Review by esauerbo — CHANGES_REQUESTED** [review]
@sj-h4 thank you for your contribution! Left a few comments.

**Comment by sj-h4** [comment]
@esauerbo 
Thank you for your review! I updated.

File: packages/ui/src/i18n/dictionaries/authenticator/ja.ts [hunk 3]
```diff
@@ -22,7 +22,7 @@ export const jaDict: AuthenticatorDictionary = {
   'Enter your code': 'コードを入力',
   'Enter your Email': 'メールアドレスを入力',
   'Enter your phone number': '電話番号を入力',
-  'Enter your username': 'ユーザー名を入力 ',
+  'Enter your Username': 'ユーザー名を入力 ',
```
Comment (reesscot) [inline packages/ui/src/i18n/dictionaries/authenticator/ja.ts:25]: @esauerbo Can we confirm that we don't use both lowercase and uppercase versions of this string?

**Review by zchenwei — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

**Pull request merged**